### ISSUE TITLE
GGMax Bug - DEVTEST - Editor Unstable

### ISSUE THREAD

**Issue by OldFlak** [TheGameCreators/GameGuruRepo#5692]
In recent updates the editor has become increasingly unstable.

As of today, particularly when editing terrain, MAX very easily crashes to desktop.

Also getting frequent momentary lockups when moving throughout the level.

All drivers are up to date, and MAX is verified via Steam

12th Gen Intel(R) Core(TM) i7-12700KF   3.61 GHz
RTX3070 Ti
32.0 GB (31.8 GB usable)
Windows 11 Pro

**Comment by Necrym59** [comment]
Agreed I am also having a few issues similar with lockups and occasional crashes to desktop
Intel(R) Core(TM) i9-10900K CPU 3.70 GHz
32.0 GB
RTX3080
Windows 11 Pro

**Comment by SampeyFS** [comment]
Exactly the same thing I had also reported

**Comment by SampeyFS** [comment]
https://github.com/TheGameCreators/GameGuruRepo/issues/5650

**Comment by LeeBamberTGC** [comment]
@OldFlak How about the latest DEVTEST build. I will also upload a new one later in case I have fixed something since the last one.

**Comment by OldFlak** [comment]
@LeeBamberTGC sorry for delay - kinda stretched for time atm
Current build seems fine at the moment 👍

**Comment by OldFlak** [comment]
Reopening this, as recent builds have seen this issue return.

The difference is that now MAX freezes for anything from 2-5 or even more quite frequently.

**Comment by LeeBamberTGC** [comment]
@OldFlak Are you able to catch this on video, the 3 to 5 second delay when using the level editor?

**Comment by SampeyFS** [comment]
yes true rightly so

**Comment by OldFlak** [comment]
Hi @LeeBamberTGC, I think it may be fixed in current build - I will give a good hammering tonight after work to be sure :)

Thanks for your efforts Lee :)

**Comment by Kasseyus** [comment]
As this ticket has received no updates in over 30 days, I will temporarily mark the issue as closed.  Should you find that the problem is unresolved or re-emerges, please let me know by updating this ticket and I will investigate further.

**Comment by Kasseyus** [comment]
The DevTest branch receives numerous updates quite regularly, so issues like this may arise from time to time.

When you experience a crash event, is there a common factor that you can think of, such as editing terrain, using the Building Editor, etc.

Also, the next time there is a crash, would you be able to post the log files as this may help pinpoint what is causing the crash to occur.  These can be found in the GameGuruMAX writables folder, Guru-Game.log and Guru-MapEditor.log.
